cfpb/consumerfinance.gov
# Ask CFPB: Move language link to standard translation-links.html template

Adds override for `get_translation_links` so that we can show the standard translation-links.html module on ask answer pages.

## Changes

- Ask CFPB: Move language link to standard translation-links.html template


## How to test this PR

1. Visit an ask answer page and see that there're translation links you can toggle between:
http://localhost:8000/ask-cfpb/a-deposit-was-credited-to-my-account-by-mistake-can-the-bankcredit-union-take-it-back-en-1015/

2. On the config page in Wagtail of the Spanish Ask CFPB landing page, you'll want to point to the english Ask CFPB page, then visit the ask landing page and there should be language links
http://localhost:8000/ask-cfpb/

## Screenshots

Before:
<img width="1285" alt="Screenshot 2024-01-19 at 3 11 47 PM" src="https://github.com/cfpb/consumerfinance.gov/assets/704760/17f1bcac-9837-42ce-918c-012109616e00">

After:
<img width="1310" alt="Screenshot 2024-01-19 at 3 11 31 PM" src="https://github.com/cfpb/consumerfinance.gov/assets/704760/35e6b3b3-2f1a-494f-96bd-8671bc295121">

Before:
<img width="1269" alt="Screenshot 2024-01-19 at 2 44 27 PM" src="https://github.com/cfpb/consumerfinance.gov/assets/704760/20758e21-3e52-4733-a398-c05aa670e59b">

After:
<img width="1263" alt="Screenshot 2024-01-19 at 2 44 15 PM" src="https://github.com/cfpb/consumerfinance.gov/assets/704760/3663280e-ef65-4e52-bd47-251e44b13c3b">
